recipes/setuptools/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version="20.7.0" %}
+
+package:
+  name: setuptools
+  version: 20.7.0
+
+source:
+  fn: setuptools-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/setuptools/setuptools-{{ version }}.tar.gz
+  md5: 5d12b39bf3e75e80fdce54e44b255615
+  patches:
+    # modify setuptools to barf if used in conda build (encourage people to add all deps in meta.yaml)
+    - nodownload.patch
+    # Don't install certifi or wincertstore
+    - setup.patch
```

Comment:
I have opened this issue ( https://github.com/pypa/setuptools/issues/668 ) upstream to hopefully provide us a way out of this without patching.